torch/csrc/jit/operator.cpp

```diff
@@ -417,6 +418,21 @@ struct OperatorRegistry  {
       return it->second;
     return empty;
   }
+
+  std::multimap<int64_t, OperatorEntry> findSimilarOperators(Symbol input_op) {
```

Comment:
why `std::multimap` over `std::vector`? It looks like the keys aren't actually used anywhere.

Reply:
So that it prints out the closer suggestions first